packages/lab/stories/form-field-next/form-field-next.stories.tsx

```diff
@@ -50,154 +50,166 @@ export const Disabled: ComponentStory<typeof FormFieldNext> = (props) => {
+        <FormHelperText>This field has been disabled</FormHelperText>
       </FormFieldNext>
       <FormFieldNext disabled {...props}>
-        <Label>Disabled Form Field</Label>
+        <FormLabel>Disabled Form Field</FormLabel>
         <InputNext variant="secondary" defaultValue="Secondary Input value" />
-        <HelperText>This field has been disabled</HelperText>
+        <FormHelperText>This field has been disabled</FormHelperText>
       </FormFieldNext>
     </FlowLayout>
   );
 };
 
-export const LabelPlacement: ComponentStory<typeof FormFieldNext> = (props) => {
+export const HelperText: ComponentStory<typeof FormFieldNext> = (props) => {
+  return (
+    <FormFieldNext style={{ width: "200px" }} {...props}>
+      <FormLabel>Form Field label</FormLabel>
+      <InputNext variant="secondary" defaultValue="Value" />
+      <FormHelperText>
+        Helper text to give more information on the field that wraps over two
+        lines
+      </FormHelperText>
+    </FormFieldNext>
+  );
+};
+
+export const Label: ComponentStory<typeof FormFieldNext> = (props) => {
   return (
     <FlowLayout style={{ width: "366px" }}>
       <FormFieldNext {...props}>
-        <Label>Form Field label top (default)</Label>
-        <InputNext defaultValue="Value" />
-        <HelperText>Default helper text</HelperText>
+        <FormLabel>Form Field label top (default)</FormLabel>
+        <InputNext variant="secondary" defaultValue="Value" />
+        <FormHelperText>Helper text</FormHelperText>
       </FormFieldNext>
       <FormFieldNext labelPlacement="left" {...props}>
-        <Label>Form Field label left</Label>
-        <InputNext defaultValue="Value" />
-        <HelperText>Default helper text</HelperText>
+        <FormLabel>Form Field label left</FormLabel>
+        <InputNext variant="secondary" defaultValue="Value" />
+        <FormHelperText>Helper text</FormHelperText>
       </FormFieldNext>
     </FlowLayout>
   );
 };
 
-export const LongLabelPlacement: ComponentStory<typeof FormFieldNext> = (
-  props
-) => {
+export const LongLabel: ComponentStory<typeof FormFieldNext> = (props) => {
   return (
     <FlowLayout style={{ width: "366px" }}>
       <FormFieldNext {...props}>
-        <Label>
+        <FormLabel>
           Form Field label that's extra long. Showing that labels wrap around to
           the next line.
-        </Label>
+        </FormLabel>
         <InputNext defaultValue="Primary Input value" />
       </FormFieldNext>
       <FormFieldNext labelPlacement="left" {...props}>
-        <Label>
+        <FormLabel>
           Form Field label that's extra long. Showing that labels wrap around to
           the next line.
-        </Label>
+        </FormLabel>
         <InputNext defaultValue="Primary Input value" />
       </FormFieldNext>
       <FormFieldNext {...props}>
-        <Label>Default Form Field label</Label>
+        <FormLabel>Form Field label</FormLabel>
```

Comment:
this example seems to be more of a long helper text example. is it necessary?

Reply:
I'd combine Label and Long Label. what do you think?

Reply:
I think this makes sense. We're not actually using this example on site, since we don't want to recommend long labels. It'll just be an example in storybook so we know how it behaves if a consumer does do it.